common/search/src/test/scala/weco/api/search/elasticsearch/ElasticsearchServiceTest.scala

```diff
@@ -105,6 +115,24 @@ class ElasticsearchServiceTest
       }
     }
 
+    it("fails if deserialising to a different type than stored") {
+      val thingsToIndex = 0.to(randomInt(5, 10)).map(_ => randomThing).toList
```

Comment:
I'm not actually sure if it's clearer, but we have a helper for this sort of construction in scala-libs:

```suggestion
      val thingsToIndex = collectionOf(min = 5, max = 10) { randomThing }.toList
```

(Possibly if that returned a List[T]?)

Reply:
It doesn't return a `List` but I do think it's clearer.